src/plays/youtube-clone/VideoViewer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import SearchBar from './SearchBar';
+import youtube from './youtube';
+import VideoList from './VideoList';
+import VideoDetail from './VideoDetail';
+class VideoViewer extends React.Component {
+    componentDidMount() {
+        this.handleSearchValue('Best books')
+    }
+    state = { apiResponse: [], selectedVideo: null }
+    handleSearchValue = async (value) => {
+        const response = await youtube.get('/search', {
+            params: {
+                q: value
+            }
+        })
+        this.setState({ apiResponse: response.data.items, selectedVideo: response.data.items[0] })
+    }
+    onVideoSelect = (video) => {
+        this.setState({ selectedVideo: video })
+    }
+    render() {
+        return (
+            <div className='ui container' style={{ marginTop: '10px' }}>
+                <SearchBar handleSearch={this.handleSearchValue} />
+                <div className='ui grid'>
+                    <div style={{display:'flex'}} className='ui row'>
+                        <div style={{height:'500px',marginTop:'10px'}} className='eleven wide column'>
+                            {this.state.selectedVideo ? <VideoDetail video={this.state.selectedVideo} /> : null}
+                        </div>
+                        <div className='five wide column'>
```

Comment:
Where do you use these class names in your stye? Also, are you sure to use 3 class selectors here?

Reply:
Actually I am using semantic UI cdn. that is the reason I gave those classnames

Reply:
Can we use the style written by us than semantic CSS? It is because We have decided to use the following style system support,

- SASS
- Raw CSS
- TailwindCSS(Upcoming)

I wonder if the entire app is not going for the semantic CSS, there are higher chances of overriding. I think, for this app, it may be easy to just write or own CSS or SASS.